Src/Support/Google.Apis.Auth.AspNetCore/GoogleScopedAuthorizeHandler.cs

```diff
@@ -0,0 +1,58 @@
+﻿/*
+Copyright 2018 Google Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Google.Apis.Auth.AspNetCore
+{
+    internal class GoogleScopedAuthorizeHandler : AuthorizationHandler<GoogleScopedRequirement>
```

Comment:
GoogleScopedAuthorizationHandler to match the base class?

Reply:
Done